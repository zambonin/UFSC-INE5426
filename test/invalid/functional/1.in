# assignment to wrong type
int a = 2, b = 4, c
lambda int x, y -> x & y
c = Î»(a, b)
