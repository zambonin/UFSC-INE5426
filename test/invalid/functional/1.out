[Line 5] semantic error: attribution operation expected integer but received boolean
